cypress/e2e/awx/access/teams.cy.ts

```diff
@@ -87,16 +87,6 @@ describe('teams', () => {
     cy.clickButton(/^Close$/);
   });
 
-  it('edit team', () => {
-    cy.navigateTo(/^Teams$/, true);
-    cy.clickRow(team.name);
-    cy.clickButton(/^Edit team$/);
-    cy.hasTitle(/^Edit team$/);
-    cy.typeByLabel(/^Name$/, 'a');
-    cy.clickButton(/^Save team$/);
-    cy.hasTitle(`${team.name}a`);
-  });
-
   it('team details', () => {
```

Comment:
while this is much like the edit from details page, deleting this gets rid of the coverage on clicking the row edit going to the edit page. in addition, edit from details returns to the team details. edit from row, returns to the teams page.